src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -57,25 +57,27 @@ interface IEigenPodManager is IPausable {
         /// @dev The relay does not additionally check any property of this digest, but needs the
         /// digest in order to reconstruct the ReceiptMetadata hash to which the proof is linked.
         bytes32 postStateDigest;
-        //journal + imageID generated by offchain proof
-        bytes payload;
+        //journal generated by offchain proof
+        Journal journal;
+        // imageID generated by offchain proof
+        bytes32 imageId;
     }
 
     struct Journal {
         // amount being proven for withdrawal
-        uint64 provenPartialWithdrawalSumGwei;
+        uint64[] provenPartialWithdrawalSumsGwei;
         // computed blockRoot
         bytes32 blockRoot;
         // the address of the pod being proven for
-        address podAddress;
+        address[] podAddresses;
         // the address of the pod owner
-        address podOwner;
+        address[] podOwners;
         // the latest timestamp proven until
-        uint64 mostRecentWithdrawalTimestamp;
+        uint64[] mostRecentWithdrawalTimestamps;
         // upper bound of the withdrawal period
-        uint64 endTimestamp;
+        uint64[] endTimestamps;
```

Comment:
another approach could be to have a consistent end timestamp for all proofs determined by backend (i.e. next end timestamp is every 12 hrs). ask @taekyunggg about the UX of each

Reply:
oops nvm this doesn't work because they don't know how to set the min proven sum in this case